src/main/java/com/google/sps/servlets/NewRequestServlet.java

```diff
@@ -201,12 +228,25 @@ private Date parseDate(String dateString) {
   /**
    * Returns the integer value of numberString
    */
-  private int parseNumber(String numberString) {
+  private Integer parseInteger(String numberString) {
```

Comment:
Not sure why.

If the fields are not set (e.g. request.getParameter("latitude") is empty), we can fallback to parsing from address.
If the user passes in wrong data (request.getParameter("latitude") is "blabla"), we should probably just return an error as opposed to silently falling back to the address.

Reply:
The latitude and longitude parameters are hidden to the user. They are set from javascript using Geolocation if the user checks the` use-current-address` checkbox. If the user didn't allow access to his current location or he wrote a different start address in the `start-address` text area, then that address should be used.
 
I wanted to store latitude and longitude so that the request to Geocoding API is made only when the user sets the address himself. In this case, is it bad practice to store both the address and the coordinates (latitude and longitude)?